spec/unit/http-api/utils.spec.ts

```diff
@@ -93,6 +93,9 @@ describe("parseErrorResponse", () => {
                     getResponseHeader(name: string): string | null {
                         return name === "Content-Type" ? "application/json" : null;
                     },
+                    getAllResponseHeaders() {
+                        return "Content-Type: application/json";
```

Comment:
NB that XHR's getAllResponseHeaders is specced to lower-case the header names

```suggestion
                        return "content-type: application/json";
```

Reply:
The refactored tests avoid case-sensitivity issues.